src/std/net/s3/api.ss

```diff
@@ -0,0 +1,242 @@
+;;; -*- Gerbil -*-
+;;; (C) vyzo
+;;; AWS S3 client
+(import "sigv4"
+        :std/net/request
+        :std/misc/func
+        :std/contract
+        :std/net/uri
+        :std/crypto/digest
+        :std/text/hex
+        :std/xml
+        :std/error
+        :std/sugar
+        :std/srfi/19)
+(export (struct-out s3-client bucket) S3ClientError)
```

Comment:
let's not export the struct, just the constructor that wraps the interface.

Reply:
Do you have an example I can reference? Where do you want that?

Reply:
Just add a constructor and export that.